internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,70 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur"}
```

Comment:
```suggestion
	excludeDirs := []string{".zeabur"}
```

Reply:
This is delete after docker build，so it is necessary to delete .git dirctory.

Reply:
Okay.